tests/end_to_end_test/snapshot_fail_test.go

```diff
@@ -283,7 +283,7 @@ func testPermissions(e *testenv.CLITest, t *testing.T, source, modifyEntry, rest
 }
 
 func parseSnapID(t *testing.T, lines []string) string {
-	pattern := regexp.MustCompile(`Created snapshot with root [\S]+ and ID ([\S]+) in .*`)
+	pattern := regexp.MustCompile(`Created snapshot with root \S+ and ID (\S+) in .*`)
```

Comment:
@redgoat650 PTAL

Reply:
FWIW this change was suggested by linter, since the test is not failing it must be good.

Reply:
Yeah this is an equivalent expression. Tested manually as well, just in case